src/frontend/src/pages/shared-components/appbar.tsx

```diff
@@ -28,13 +20,11 @@ export const ApplicationBar: React.FC = () => {
                                 </IconButton>
                             </Grid>
                             <Grid item>
-                                <Typography display="inline">
-                                    Logged in As Tina
-                                </Typography>
+                                <Typography display="inline">Username</Typography>
                             </Grid>
                             <Grid item>
                                 <Box p={3}>
-                                    <Typography variant="h6" display="inline">
+                                    <Typography variant="subtitle1" display="inline">
```

Comment:
The other inline element "Username" doesn't have a variant so this shouldn't either. 
```suggestion
                                    <Typography display="inline">
```